packages/devel/flex/patches/c5a26b17a2b091c560f6c4e6703e55f496bf74fa.patch

```diff
@@ -0,0 +1,28 @@
+From c5a26b17a2b091c560f6c4e6703e55f496bf74fa Mon Sep 17 00:00:00 2001
```

Comment:
What is this patch about? Where is it from? Why is it needed?

Reply:
Patch is a upstream 2.6.5 patch. Referenced from https://github.com/westes/flex/pull/262 https://github.com/westes/flex/issues/241 

without the patch the build SEG faults. Is included in master
```
./stage1flex   -o stage1scan.c /home/rudi/LibreELEC.tv/build.LibreELEC-Generic.x86_64-9.80-devel/build/flex-2.6.4/src/scan.l
make[2]: *** [Makefile:1689: stage1scan.c] Segmentation fault
```